src/current/v23.2/show-statements.md

```diff
@@ -28,7 +28,7 @@ In CockroachDB, the following are aliases for `SHOW STATEMENTS`:
 
 ## Required privileges
 
-All users can see their own currently active queries. All users belonging to the `admin` role can view all users' currently active queries.
+All users can see their own currently active queries. Users with the VIEWACTIVITY or VIEWACTIVITYREDACTED privilege can view all users' currently active queries. VIEWACTIVITYREDACTED causes constants in queries being executed by other users to be redacted.
```

Comment:
same as above

Can you please change

```
Users with the VIEWACTIVITY or VIEWACTIVITYREDACTED privilege
```

to

```
Users with the [`VIEWACTIVITY` or `VIEWACTIVITYREDACTED` privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges)
```

and add backticks around the second use of `VIEWACTIVITYREDACTED` ?